views/layouts/layout-pane.njk

```diff
@@ -1,22 +1,30 @@
 <!doctype html>
-<html>
+<html class="no-js">
   <head>
     <meta charset="utf-8">
     <meta name="robots" content="noindex, nofollow">
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <title>{{ title }}</title>
     <link href="/stylesheets/main.css" rel="stylesheet" media="all" />
-    <!--[if lt IE 9]><script src="/javascripts/ie.js"></script><![endif]-->
-    <script src="/javascripts/application.js"></script>
+    <!--[if lt IE 9]>
+      <script src="/javascripts/ie.js"></script>
+    <![endif]-->
+    <script src="javascripts/vendor/modernizr.js"></script>
```

Comment:
Needs a leading slash – getting a 404 for this on pages other than the homepage.

Reply:
Done!